python/common/org/python/stdlib/datetime/TimeDelta.java

```
@@ -178,6 +232,56 @@ public TimeDelta(org.python.Object[] args, java.util.Map<java.lang.String, org.p
         return TD;
     }
 
+    @org.python.Method(__doc__ = "", args = {"other"})
+    public org.python.Object __sub__(org.python.Object other) {
+        long thisDays = ((org.python.types.Int) this.days).value;
+        TimeDelta otherObject = (org.python.stdlib.datetime.TimeDelta) other;
```

This could throw a `ClassCastException` but it should for example instead throw `TypeError: unsupported operand type(s) for -: 'datetime.timedelta' and 'str'`

---

Otherwise it looks good :+1:

---

Fixed for both add and sub ➕➖